tests/acceptance/helpers/sharingHelper.js

```diff
@@ -67,38 +67,32 @@ module.exports = {
     return fetch(apiURL, { method: 'GET', headers: headers })
       .then(res => res.json())
       .then(function (sharesResult) {
-        if (sharesResult.ocs.meta.statuscode === 200) {
-          const shares = sharesResult.ocs.data
-          let found
-          for (var shareI = 0; shareI < shares.length; shareI++) {
-            const share = shares[shareI]
-            found = true
-            for (var expectedDetailsI = 0; expectedDetailsI < expectedDetailsTable.hashes().length; expectedDetailsI++) {
-              const expectedDetail = expectedDetailsTable.hashes()[expectedDetailsI]
-              if (expectedDetail.field === 'permissions') {
-                expectedDetail.value = sharingHelper.humanReadablePermissionsToBitmask(expectedDetail.value).toString()
-              } else if (expectedDetail.field === 'share_type') {
-                expectedDetail.value = sharingHelper.humanReadableShareTypeToNumber(expectedDetail.value).toString()
-              }
-              if (!(expectedDetail.field in share) || share[expectedDetail.field].toString() !== expectedDetail.value) {
-                found = false
-                break
-              }
+        httpHelper.checkOCSStatus(sharesResult, 'Could not get shares. Message: ' + sharesResult.ocs.meta.message)
```

Comment:
Why was this function changed?

Reply:
the function is not completely changed! Some changes had to be made like using checkOCSStatus function to check response and for loop pattern etc